crates/primitives/src/specification.rs

```diff
@@ -62,6 +62,7 @@ pub enum SpecId {
     CANYON = 19,
     CANCUN = 20,
     ECOTONE = 21,
+    FERMAT = 22,
```

Comment:
Please change this PR from draft to R4R, and improve the PR description

Reply:
updated